.changeset/khaki-pugs-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-notifications': minor
```

Comment:
```suggestion
'@backstage/plugin-notifications': patch
```
Package is below 1.x so a `patch` changeset is enough for introducing new non breaking features.